.github/workflows/tests_smoke.yml

```diff
@@ -0,0 +1,31 @@
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Test suite
+
+on:
+  pull_request:
+    types: [ opened, synchronize, reopened, ready_for_review ]
```

Comment:
I wonder if the `ready_for_review` type would be needed with this new approach. Every new commit, no matter the PR is a draft or ready for review, will get these tests triggered. Why would we want to re-trigger the tests when the PR is moved from `Draft` to `Ready for review` (if no new commits exist since last run) ?

Reply:
No we do not, I will remove it.